src/views/ManageAddress/index.tsx

```diff
@@ -17,13 +19,19 @@ import TranslatedText from 'components/TranslatedText'
 import { usePanelContext } from 'contexts/panelContext'
 import { Page, routes } from 'routes'
 
+export enum Language {
```

Comment:
Please move this enum to some common ground (what do you think about introducing `common` directory?) It should be used in the entire application and is not specifically bonded to place.